### ISSUE TITLE
NPE in MonocleWindowManager with development version of plugin

### ISSUE THREAD

**Issue by mipastgt** [gluonhq/client-maven-plugin#273]
Hi,
with the latest development version of the client-maven-plugin (0.1.34-SNAPSHOT which I just built myself) I get a NullPointerException in MonocleWindowManager when I build and run an app on Android. The screen on the device is then garbled too. See attached screenshot.

```
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] V/GraalActivity(21393): surfaceChanged start
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] V/GraalActivity(21393): [MainActivity] surfaceChanged, format = 4, width = 2560, height = 1344
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] E/GraalGluon(21393): nativeSetSurface called, env at 0x7dbb0d6200 and size 8, surface at 0x7fd78ac208
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] E/GraalGluon(21393): native setSurface Ready, native window at 0x7da83c1010
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] V/GraalActivity(21393): surfaceChanged done, metrics = DisplayMetrics{density=4.0, width=2560, height=1440, scaledDensity=4.0, xdpi=580.571, ydpi=575.433}, density = 4.0
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] V/GraalActivity(21393): SurfaceRedraw needed start
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] V/GraalActivity(21393): ask native graallayer to redraw surface
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] E/GraalGluon(21393): launcher, nativeSurfaceRedrawNeeded called. Invoke method on glass_monocle
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): traceEvent: Set MouseState[x=320,y=168,wheel=0,buttonsPressed=IntSet[]]
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): Exception in thread "JavaFX Application Thread" java.lang.NullPointerException
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.sun.glass.ui.monocle.MonocleWindowManager$1.run(MonocleWindowManager.java:179)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at java.security.AccessController.doPrivileged(AccessController.java:101)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.sun.glass.ui.monocle.RunnableProcessor.runLoop(RunnableProcessor.java:92)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.sun.glass.ui.monocle.RunnableProcessor.run(RunnableProcessor.java:51)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at java.lang.Thread.run(Thread.java:834)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.oracle.svm.core.thread.JavaThreads.threadStartRoutine(JavaThreads.java:517)
[Wed Oct 21 14:49:24 CEST 2020][INFO] [SUB] D/GraalCompiled(21393): 	at com.oracle.svm.core.posix.thread.PosixJavaThreads.pthreadStartRoutine(PosixJavaThreads.java:192)
[
```
This was tested with my Samsung Galaxy S7 phone with Android 8 and the unmodified HelloFX app from the gluon-samples. Just incremented the version number of the client-maven-plugin in the POM.

![Screenshot_20201021-144917](https://user-images.githubusercontent.com/16735714/96722017-34bcf200-13ad-11eb-9d51-0d381e56c813.jpg)

**Comment by jperedadnr** [comment]
See https://github.com/gluonhq/substrate/issues/810
It should be fixed soon, the regression in OpenJFX has been recently fixed.

**Comment by jperedadnr** [comment]
Should be fixed now with 0.1.34-SNAPSHOT

**Comment by mipastgt** [comment]
My Linux machine just died, so I cannot test it out at the moment until I have that fixed or replaced :-(
